LearningMissionLab/Schedule.cs

```diff
@@ -1,10 +1,26 @@
 ﻿using System;
+using System.Collections.Generic;
+
 namespace LearningMissionLab
 {
     public class Schedule
     {
-        public Schedule()
+        string _zoomReferanse;
```

Comment:
You have a typo here, it should be zoomReference. However, the zoom meeting property is more interesting. I think you should consider adding a helper class for it, ZoomMeeting with three fields: link, id, passcode, like in the following example:

Link: https://us04web.zoom.us/j/6323504871?pwd=SjZoK0djS0FFWFlKZDFNdHY2T0F
Meeting ID: 632 350 4871
Passcode: 9hUJ8X